src/webui/api/authcontroller.cpp

```diff
@@ -45,29 +46,41 @@ void AuthController::loginAction()
         return;
     }
 
-    if (isBanned())
+    const QString clientAddr {sessionManager()->clientId()};
+    const QString usernameFromWeb {params()["username"]};
+    const QString passwordFromWeb {params()["password"]};
+
+    if (isBanned()) {
+        LogMsg(tr("WebUI login failure. Reason: IP has been banned, IP: %1, username: %2")
```

Comment:
Now it is not a web-only API, but some generic API, independent of the web (although some forgotten references may still be left). The WebAPI is one of possible wrappers around it. There are no WebUI, IP etc. inside it. So please don't use these terms here.

I thought about its name, but I didn't approve it completely. It could be something like SerialAPI (SAPI) since it uses serialized data as input/output. What do you say?

Reply:
Without telling it is WebUI | WebAPI, how could user recognize what is happening & from where?
The log message is provided for both user & developer, so if we use SAPI and the like, what will user think? will they be able to (easily) understand what is going on?
In this case, I think log messages that are easier (for user) to understand is more important than faithfully reflecting the nature of underlying code structure.

Also "WebAPI" is a bit more generic and I can change to it if requested.

Reply:
>It's about application design. Imagine, tomorrow we will add a new wrapper (ex., DBus or other Protocol) and we will get confusing messages in the log.

I consider going that way in the future is unlikely...

>Then we need to produce more generic messages, or handle it on different level.

I think what you want is define an specific function like `LogMsg()` (for internal use in class) but at the base class, which will automatically prepends the specific use condition, e.g. "WebAPI" or something else in the log message.
I suppose it can be in apicontroller? or any other place you suggest?

Reply:
I can't do this right now. So I can approve the current version as a temporary solution if you see it as important to merge it sooner. But you should still use WebAPI instead of WebUI.